apps/golden-sample-app/src/app/app.component.html

```diff
@@ -80,7 +78,6 @@
           [attr.aria-current]="
             transactionsListRouterLink.isActive ? 'page' : undefined
           "
-          role="link"
```

Comment:
Why did you remove all these "role"?

Reply:
Because it is not needed. Html tag anchor has `role="link"` by default.